Revise known issues for Logstash 9.3.0 and 9.2.5 (#18800)
elastic/logstash: docs/release-notes/known-issues.md

## 9.3.0 [logstash-ki-9.3.0]

**Logstash will not start with bundled JDK on aarch64 Linux and x86_64 Windows**

Applies to: {{ls}} 9.3.0

::::{dropdown} Details

All {{ls}} 9.3.0 artifacts were bundled with Linux x86_64 JDK due to a bug in artifact generation.

> Starting {{ls}} on Windows or aarch64 Linux/MacOS with 9.3.0 artifacts will result in an fatal error as the bundled JDK will not execute on those platforms.

Reviewer: ```suggestion
Starting {{ls}} on Windows or aarch64 Linux/MacOS with 9.3.0 artifacts results in an fatal error as the bundled JDK is not compatible with those platforms.
```

This issue affects:
* logstash-9.3.0-linux-aarch64.tar.gz
* logstash-9.3.0-darwin-aarch64.tar.gz
* logstash-9.3.0-windows-x86_64.zip
* logstash-9.3.0-arm64.deb
* logstash-9.3.0-aarch64.rpm

This issue does NOT affect:
* logstash-9.3.0-linux-x86_64.tar.gz
* logstash-9.3.0-x86_64.deb
* logstash-9.3.0-x86_64.rpm
* Docker images

For this reason the download links to the affected artifacts have been removed from the Download Page of {{ls}} 9.3.0.

As a workaround, users can provide an external and compatible JDK via the `LS_JAVA_HOME` environment variable.

::::

## 9.2.5 [logstash-ki-9.2.5]

> **Logstash will not start with bundled JDK on aarch64 Linux and x86_64 Windows**

Reviewer: ```suggestion
**Logstash does not start with bundled JDK on aarch64 Linux and x86_64 Windows**
```

Reviewer: Present tense to match style.

Applies to: {{ls}} 9.2.5

::::{dropdown} Details

All {{ls}} 9.2.5 artifacts were bundled with Linux x86_64 JDK due to a bug in artifact generation.
Starting {{ls}} on Windows or aarch64 Linux/MacOS with 9.2.5 artifacts will result in an fatal error as the bundled JDK will not execute on those platforms.

This issue affects:
* logstash-9.2.5-linux-aarch64.tar.gz
* logstash-9.2.5-darwin-aarch64.tar.gz
* logstash-9.2.5-windows-x86_64.zip
* logstash-9.2.5-arm64.deb
* logstash-9.2.5-aarch64.rpm

This issue does NOT affect:
* logstash-9.2.5-linux-x86_64.tar.gz
* logstash-9.2.5-x86_64.deb
* logstash-9.2.5-x86_64.rpm
* Docker images

> For this reason the download links to the affected artifacts have been removed from the Download Page of {{ls}} 9.2.5.

Reviewer: ```suggestion
For this reason the download links to the affected artifacts have been removed from the [{{ls}} Download page](https://www.elastic.co/downloads/logstash) for version 9.2.5.
```
